topdown/strings_test.go

```diff
@@ -0,0 +1,93 @@
+package topdown
```

Comment:
One last nit: Can you add the copyright header into the new file?

Reply:
94ba0cf addresses this.